pkg/configs/default.go

```diff
@@ -0,0 +1,38 @@
+package configs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/jetstack/preflight/pkg/agent"
+	"gopkg.in/yaml.v2"
+)
+
+func GetDefaultDataGatherers() ([]agent.DataGatherer, error) {
```

Comment:
Please add some documentation for your functions, in the format of `// GetDefaultDataGatherers ...`